nautobot_design_builder/jobs.py

```diff
@@ -27,14 +31,29 @@ class Meta:  # pylint: disable=too-few-public-methods
     def run(self, data, commit):
         """Execute Decommissioning job."""
         deployments = data["deployments"]
+        only_traceability = data["only_traceability"]
```

Comment:
We need a different name than `only_traceability`. It is not clear what the meaning of that variable is without really digging into the code. I'll give this some thought.